data/json/monsters/misc.json

```diff
@@ -211,5 +211,34 @@
     "special_attacks": [ [ "DISAPPEAR", 200 ] ],
     "death_function": { "message": "The %s disappears.", "corpse_type": "NO_CORPSE" },
     "flags": [ "FLIES", "NO_BREATHE", "NOT_HALLUCINATION", "HARDTOSHOOT" ]
+  },
+  {
+    "id": "mon_mannequin_decoy",
+    "type": "MONSTER",
+    "name": { "str": "mannequin decoy" },
+    "description": "A human sized mannequin, making noise.",
+    "looks_like": "mon_shia",
+    "default_faction": "factionless",
+    "species": [ "UNKNOWN" ],
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 100,
```

Comment:
This is only going to take a hit or two before it falls apart, 100 hp is way too high.

Reply:
It's the size of an adult and made of wood, shouldn't it last a few minutes?

Also, could you recommend a specific number? I have not yet developed a sense for these values.

Reply:
Probably about 20 hp